roles/confluent.common/defaults/main.yml

```diff
@@ -46,3 +46,12 @@ confluent_cli_download_enabled: "{{rbac_enabled}}"
 
 ### Full path on hosts to install the Confluent CLI
 confluent_cli_path: /usr/local/bin/confluent
+
```

Comment:
Hey, great work adding docs to VARIABLES.md. And this was not mentioned to you yet, but the way that file works is if you add an annotation to the variable here like 

```
#### Base URL for Confluent's Archive Repositories
confluent_common_repository_archive_baseurl: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}"
```

and then run `python docs.py`

then it will "write" the VARIABLES.md file for you. ie its self documenting. Can you please add annotations and proceed that way?

Reply:
Done and I regenerated `VARIABLES.md`